### DISCUSSION TITLE
Bundler option for `nx/nest` apps

### DISCUSSION THREAD

**Discussion by raoulkent** [nrwl/nx#29263] [Feature Requests]
Currently there is no option other than webpack for generating Nest applications.
This leads to some annoyances where one must generate with webpack, then do manual changes to switch to something else.

It would be preferable if this were instead possible to generate `[none, 'webpack', 'esbuild']` or somesuch.

**Comment by db-lyon** [comment]
wow. this is extremely disappointing.

**Comment by kowalk66** [comment]
just had similar issue now. I fully agree that a lot of people who are using nx are having nest in the workspace, that would fully benefit a lot of users having more options in nest generator.
